gluoncv/data/tracking_data/track.py

```diff
@@ -0,0 +1,537 @@
+        s_z = np.sqrt(wc_z * hc_z)
+        scale_z = exemplar_size / s_z
+        w = w*scale_z
+        h = h*scale_z
+        cx, cy = imw//2, imh//2
+        bbox = center2corner(Center(cx, cy, w, h))
+        return bbox
+
+    def __len__(self):
+        return self.num
+
+    def __getitem__(self, index):
+        index = self.pick[index]
+        data, index = self._find_dataset(index)
+
+        gray = self.gray and self.gray > np.random.random()
+        neg = self.neg and self.neg > np.random.random()
+
+        # get one dataset
+        if neg:
+            template = data.get_random_target(index)
+            search = np.random.choice(self.all_dataset).get_random_target()
+        else:
+            template, search = data.get_positive_pair(index)
+
+        # get image
+        template_image = self.cv2.imread(template[0])
+        search_image = self.cv2.imread(search[0])
+
+        # get bounding box
+        template_box = self._get_bbox(template_image, template[1])
+        search_box = self._get_bbox(search_image, search[1])
+
+        # augmentation
+        template, _ = self.template_aug(template_image,
+                                        template_box,
+                                        self.train_exemplar_size,
+                                        gray=gray)
+
+        search, bbox = self.search_aug(search_image,
+                                       search_box,
+                                       self.train_search_size,
+                                       gray=gray)
+
+        # get labels
+        cls, delta, delta_weight, _ = self.anchor_target(bbox, self.train_output_size, neg)
+        template = template.transpose((2, 0, 1)).astype(np.float32)
+        search = search.transpose((2, 0, 1)).astype(np.float32)
+
+        return template, search, cls, delta, delta_weight, np.array(bbox)
+
+class AnchorTarget:
+    def __init__(self, anchor_stride, anchor_ratios, train_search_size, train_output_size):
+        """create anchor target
+
+        Parameters
+        ----------
+        anchor_stride : int
+            anchor stride
+        anchor_ratios : tuple
+            anchor ratios
+        train_search_size : int
+            train search size
+        train_output_size : int
+            train output size
+        """
+        self.anchor_stride = anchor_stride
+        self.anchor_ratios = anchor_ratios
+        self.anchor_scales = [8]
+        self.anchors = Anchors(self.anchor_stride,
+                               self.anchor_ratios,
+                               self.anchor_scales)
+        self.train_search_size = train_search_size
+        self.train_output_size = train_output_size
+
+        self.anchors.generate_all_anchors(im_c=self.train_search_size//2,
+                                          size=self.train_output_size)
+        # Positive anchor threshold
+        self.train_thr_high = 0.6
+        # Negative anchor threshold
+        self.thain_thr_low = 0.3
```

Comment:
Why the value here hardcoded? Is this standard for most tracking algorithm? Or just for SiamRPN? 

Because if these values will change in the future, it would be better to write them in arguments so that users can pass in these values instead of hardcode it.